src/firefly/java/edu/caltech/ipac/firefly/server/visualize/FitsCacher.java

```diff
@@ -59,9 +64,13 @@
                     else {
                         Fits fits= null;
                         try {
+                            if (memCache != null) {
+                                memCache.put(key, new ObjectSizeEngineWrapper.BluffSize(fitsFile.length()));
+                                memCache.put(key, null);
```

Comment:
This is how I am clear space in the cache before I actually read the file.  If there is a 5G file in the cache by doing this I can potentially push out the 5G from the cache before the 5G file is read. Since the act of reading would takes 10G we would go as high as 15G. With this change it will only go as high as 10G the down to 5G.

To summarize- replace a f5G file in cache:
before:
5G => 15G => 5G
after
5G => 10G => 5G

The `ObjectSizeEngineWrapper.BluffSize` object appears to the cache to be big but it is not.